api/src/api/user.js

```diff
@@ -0,0 +1,145 @@
+const express = require('express');
+const router = express.Router();
+const { errorWrap } = require('../middleware');
+
+const User = require('../models/user');
+
+router.post(
+  '/',
+  errorWrap(async (req, res) => {
+    const newUser = await User.create(req.body);
+    if (newUser) {
+      res.status(200).json({
+        message: 'Successfully created new user',
+        success: true,
+        result: newUser,
+      });
+      return;
+    }
+  }),
+);
+
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    const users = await User.find();
+    res.status(200).json({
+      message: 'Successfully retrieved users',
```

Comment:
Add 'Successfully retrieved all users' to make it clear that we're getting all users